lib/proxi/local/sdrf_handler.py

```
@@ -0,0 +1,729 @@
+    sdrf = SDRFHandler(verbose=params.verbose)
+    sdrf_cache_path = "sdrf_files"
+
+    if params.url is None and params.filepath is None:
+        eprint(f"ERROR: Either --url or --filepath must be provided. Use --help for more info")
+        return
+
+    if params.url is not None:
+        sdrf.data_url = params.url
+        sdrf.ui_url = params.url
+
+        response = requests.get(params.url)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)